ibis/sql/alchemy.py

```diff
@@ -682,6 +712,34 @@ def _ntile(t, expr):
     ops.CumulativeMean: unary(sa.func.avg),
 }
 
+if GEO:
+    # TODO: Add functions for a lot of new ops, including
+    # ST_Buffer, ST_Azimuth, ST_Centroid, ST_CoveredBy,
+    # ST_ContainsProperly, ST_CRosses, ST_Difference
+    _geospatial_functions = {
+        ops.GeoArea: unary(sa.func.ST_Area),
```

Comment:
The list of MapD supported functions and PostGIS supported functions has some significant differences. We may want to extend and rationalize the list of ops.

Reply:
you're right. AFAIK MapD has just a subset of functions that PostGIS has. 
but maybe it could be done in a follow up PR.

Reply:
For the most part I don't think it will be too hard to add these (at least a useful subset of them)